README.md

```diff
@@ -37,9 +37,18 @@
 
 <br>
 
-Strapi is a free and open-source headless CMS delivering your content anywhere you need.
 
-- **Keep control over your data**. With Strapi, you know where your data is stored, and you keep full control at all times.
+Strapi is an open-source software helping developers to easily build, deploy, and manage APIs. Its open-source headless CMS provides backend functionality for applications and gives developers the freedom to use the tools of their choice for front-end frameworks. It’s called headless because the frontend is not part of the CMS, but there’s an APIfor content to show up on different devices. Moreover, Strapi is a free and open-source headless CMS delivering your content anywhere you need.
+
+The company was founded in 2016 and headquartered in Paris, Ile-de-France.
+
+## Strapi brings unique characteristics to the headless eCommerce world that are already available in traditional websites, but only used by a very limited amount of people:
+
+- **Content Management System (CMS):** Strapi’s Admin panel is extremely intuitive and easy to use for content editors. It allows them to manage complex content structures, categories, and subcategories with simple drag & drop operations.
+- **Content Relationships:** As Strapi is using GraphQL under the hood to display content on pages (we will see this later), it’s trivial for developers to manage relations between objects of different data types within a single endpoint. For instance, you can have an article with multiple authors.
+- **Content Search:** By using the ElasticSearch tool, you can easily index your content and create powerful search features. As a consequence, it’s easier for users to find what they are looking for in a simple way.
```

Comment:
I'm not sure we would like to highlight ElasticSearch here. From my understanding, ElasticSearch support is not built-in into Strapi but can be added with a plugin.

Reply:
Yes Elasticsearch doesn't really have its place in our README :)